apps/core-v2-testnet/src/utils/getTokenIds.ts

```diff
@@ -0,0 +1,17 @@
+// Utils to get PT and YT token ids
+// Token ids reference the token values in the MultiToken contract
+
+import { BigNumber, BigNumberish } from "ethers";
+
+export function getPTId(expiry: BigNumberish): BigNumber {
+  return BigNumber.from(expiry);
+}
```

Comment:
Hmm, passing ids around as BigNumbers feels weird to me, can we use strings or numbers instead?